tools/dreport.d/openpower.d/plugins.d/vpd_data

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+#
+# config: 23 25
```

Comment:
Openpower plugin is enabled for all types openpower boxes , this is going to be issue for smaller falsh systems. already bad vpd plugin included in the existing drepaort. Not required this plugin in elog type. Also reduce the priority > 50 to accomodate high priority data . VPD data seems to be in the 100K range .